### ISSUE TITLE
With and height of bounding boxes too low but positions correct

### ISSUE THREAD

**Issue by Knuust** [AlexeyAB/darknet#3975]
Hi,
I trained a model on a bunch of 2500x2500 images. The size of input vector was 416x416. After training for 15K Iterations, I get a IoU, mAP and F1 of 0.0 for validation and training set.

When I test with sample image, the bounding boxes are at correct position but are way too small. Do you have an idea what could be the problem?
Thank you in advance!
Knust

**Comment by AlexeyAB** [comment]
Compile Darknet with OpenCV and train with flag ` -show_imgs` at the end of training command.
Do you see correct bounded boxes during training?

**Comment by Knuust** [comment]
Thank you for your reply.
I tried that and there are correct bounding boxes during training.

**Comment by AlexeyAB** [comment]
Show screenshot of mAP.
Show example of bad detection.

**Comment by Knuust** [comment]
![Screenshot from 2019-09-25 09-52-26](https://user-images.githubusercontent.com/52486814/65580864-c247cb80-df7a-11e9-9fa7-b762b3731ca2.png)
This is the mAP and 
![Screenshot from 2019-09-25 09-55-49](https://user-images.githubusercontent.com/52486814/65581002-fe7b2c00-df7a-11e9-851c-cede10f2ba35.png)
This is a test image from validation set. As you can see, the buildings are detected correctly in most cases but the bb are too small.

**Comment by AlexeyAB** [comment]
You something doing wrong.

What mAp can you get on Training dataset instead of Validation dataset?

It seems that you use different cfg-files (with different anchors) for training and detection.

**Comment by Knuust** [comment]
Yes, that was the Problem. Thank you so much for your time and help!
